adapters/Unity/UnityAdapter/GADMUnityInterstitialAd.m

```diff
@@ -0,0 +1,169 @@
+//
+//  GADMUnityInterstitialAd.m
+//  AdMob-TestApp-Local
+//
+//  Created by Kavya Katooru on 6/10/20.
+//  Copyright © 2020 Unity Ads. All rights reserved.
+//
+
+#import "GADMUnityInterstitialAd.h"
+#import "GADMAdapterUnityConstants.h"
+#import "GADUnityError.h"
+
+@interface GADMUnityInterstitialAd () <GADMAdapterUnityDataProvider, UnityAdsExtendedDelegate>
```

Comment:
Apply the same changes as for rewarded ads.

Reply:
Could you please explain the changes you meant here? Did you mean removing _uuid and impressionOrdinal etc?

Reply:
exactly, plus load api.